src/AzModuleInstaller.ts

```diff
@@ -0,0 +1,159 @@
+import * as core from '@actions/core';
+import * as tc from '@actions/tool-cache';
+import * as os from 'os';
+import { ArchiveTools } from './Utilities/ArchiveTools';
+import FileUtils from './Utilities/FileUtils';
+import Utils from './Utilities/Utils';
+
+interface InstallResult {
```

Comment:
nit. InstallationResult ?

Reply:
I don't really see a difference, will leave it this way.